app/services/power_manager/virt.rb

```diff
@@ -30,7 +30,12 @@ def state_output(result)
     end
 
     def default_action(action)
-      vm.send(action)
+      action_status = vm.send(action)
+      # Temporary fix until fog-ovirt is updated
+      if action_status.is_a?(Fog::Compute::Ovirt::Server)
+        return true
+      end
+      action_status
```

Comment:
Sorry, I take my approval back. I haven't tested, but after re-reading the code base, I see that there is a possibility for nil exception.